js/samples/06.auth.oauth.bot/env/.env.dev

```diff
@@ -0,0 +1,19 @@
+# This file includes environment variables that will be committed to git by default.
```

Comment:
Should this file be added? Or is it automatically generated by teamsfx?

Reply:
Yes, otherwise there will be no env when running the provision command.